alvr/client_openxr/src/stream.rs

```diff
@@ -94,6 +94,7 @@ pub struct StreamContext {
     target_view_resolution: UVec2,
     renderer: StreamRenderer,
     decoder: Option<(VideoDecoderConfig, VideoDecoderSource)>,
+    defer_reprojection_to_runtime: bool,
```

Comment:
To reiterate from my previous comment, we should flip the meaning of this field, to "use_custom_reprojection". Because in this way "false" should be the default.

Reply:
Resolved, renamed to `use_custom_reprojection`